test/lolex-test.js

```diff
@@ -2769,4 +2769,46 @@ describe("lolex", function () {
             }, true);
         });
     });
+
+    describe("requestIdleCallback", function () {
+
+        beforeEach(function () {
+            this.clock = lolex.createClock();
+        });
+
+        it("throws if no arguments", function () {
+            var clock = this.clock;
+
+            assert.exception(function () { clock.requestIdleCallback(); });
+        });
+
+        it("returns numeric id", function () {
+            var result = this.clock.requestIdleCallback(NOOP);
+
+            assert.isNumber(result);
+        });
+
+        it("returns unique id", function () {
+            var id1 = this.clock.requestIdleCallback(NOOP);
+            var id2 = this.clock.requestIdleCallback(NOOP);
+
+            refute.equals(id2, id1);
+        });
+
+    });
+
+    describe("cancelIdleCallback", function () {
+
+        beforeEach(function () {
+            this.clock = lolex.createClock();
+        });
+
+        it("removes idle callback", function () {
+            var stub = sinon.stub();
+            var callbackId = this.clock.requestIdleCallback(stub);
+            this.clock.cancelIdleCallback(callbackId);
+
+            assert.isFalse(stub.called);
```

Comment:
Need to call `clock.runAll` before this call in order to give it a chance to run.